packages/main/src/RPA/Desktop/keywords/finder.py

```diff
@@ -33,6 +35,50 @@ def __init__(self, ctx):
         else:
             self.confidence = None
 
+    def get_matches(self, locator: ImageTemplate) -> List[Union[Point, Region]]:
+        """Internal helper method for getting image template matches in all displays
+        and returning them as Points or Regions, scaled to accomodate macOS HiDPI
+        """
+
+        def get_scaled_matches(screenshot, locator, display):
+            """ Internal helper function for finding matches on a single screen """
+            self.logger.info("Matching with confidence of %.1f", confidence)
```

Comment:
this could be outside the looping so we don't log the same thing for every monitor